### PR TITLE
chore(website): Refactor k8s auth example

### PR DESCRIPTION
**Pull request by candiduslynx**
related to #7296

### CODE REVIEW

**Comment by candiduslynx** [comment]
@lshmouse I've updated the doc for better viewing.
Could you please validate that you achieve the same expected results as in #7296?

File: website/pages/docs/plugins/sources/k8s/overview.mdx [hunk 1]
```diff
@@ -13,20 +13,25 @@ The K8s Source plugin for CloudQuery extracts configuration from a variety of K8
...
+When `cloudquery` is syncing the Kubernetes cluster information where it is running in a pod,
+the Kubernetes Service Account can be used for direct authentication.
+
+To achieve that you need to create a cluster role with all `get` and `list` privileges:
+
+```bash copy
+kubectl apply -f - <<EOF
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
+kind:       ClusterRole
```
Comment (lshmouse) [inline website/pages/docs/plugins/sources/k8s/overview.mdx:34]: It's better to use a space after colon.

File: website/pages/docs/plugins/sources/k8s/overview.mdx [hunk 2]
```diff
@@ -43,5 +48,23 @@ rules:
...
+Then, create a cluster role binding for that role and the service account the `cloudquery` pod uses:
+
+```bash copy
+kubectl apply -f - <<EOF
+apiVersion: rbac.authorization.k8s.io/v1
+kind:       ClusterRoleBinding
+metadata:
+  name: cloudquery-cluster-read-binding
+subjects:
+- kind:     ServiceAccount
```
Comment (lshmouse) [inline website/pages/docs/plugins/sources/k8s/overview.mdx:62]: It's better to use a space after colon.

**Review by lshmouse — COMMENTED** [review]
It works after validation on my k8s cluster

File: website/pages/docs/plugins/sources/k8s/overview.mdx (hunk 1 above)
Comment (candiduslynx) [reply website/pages/docs/plugins/sources/k8s/overview.mdx:34]: only formatting (it's several spaces)

File: website/pages/docs/plugins/sources/k8s/overview.mdx (hunk 2 above)
Comment (candiduslynx) [reply website/pages/docs/plugins/sources/k8s/overview.mdx:62]: only formatting (it's several spaces)

File: website/pages/docs/plugins/sources/k8s/overview.mdx [hunk 3]
```diff
@@ -43,5 +48,23 @@ rules:
...
+  name:     cloudquery-sa
+  apiGroup: rbac.authorization.k8s.io
```
[the remark this answers is not in the document]
Comment (candiduslynx) [reply website/pages/docs/plugins/sources/k8s/overview.mdx]: Thanks! I've updated the doc, could you verify again?

**Review by hermanschaaf — APPROVED** [review]

File: website/pages/docs/plugins/sources/k8s/overview.mdx [hunk 4]
```diff
@@ -16,21 +16,25 @@ The K8s Source plugin for CloudQuery extracts configuration from a variety of K8
...
-export KUBECONFIG=<PATH_TO_YOUR_CONFIG_FILE>
+export KUBECONFIG="<PATH_TO_YOUR_CONFIG_FILE>"
 ```
 
-If `cloudquery` is running in a pod of the Kubernetes cluster whose information will be synced, the K8s service account can be used for authentication directly.
-First, create a cluster role named `cloudquery-cluster-read` with all get and list privileges of all resources.
-Then create a cluster role binding for the cluster role and the service account for the `cloudquery` pod to use.
+### Kubernetes Service Account
+
+When `cloudquery` is syncing the Kubernetes cluster information where it is running in a pod,
```
Comment (jsonpr) [inline website/pages/docs/plugins/sources/k8s/overview.mdx:29]: I like using `if` here more.   For example:

If `cloudquery` is running in a pod of the Kubernetes cluster, the Kubernetes Service Account can be used for direct authentication.  To use the Kubernetes Service Account for direct authentication, a cluster role with all `get` and `list` privileges will need to be used.

The below command creates a new cluster role with `get` and `list` privileges.

File: website/pages/docs/plugins/sources/k8s/overview.mdx [hunk 5]
```diff
@@ -47,5 +51,21 @@ rules:
   - get
   - list
 EOF
-$ kubectl create clusterrolebinding cloudquery-cluster-read-$service_account-binding --clusterrole=cloudquery-cluster-read --serviceaccount=$service_account
+```
+
+Then, create a cluster role binding for that role and the service account the `cloudquery` pod uses:
```
Comment (jsonpr) [inline website/pages/docs/plugins/sources/k8s/overview.mdx:56]: Next, the cluster role and service account will need to be linked via a cluster role binding.

The following creates a cluster role binding for the role we created above and the service account for the `cloudquery` pod.

**Review by jsonpr — COMMENTED** [review]
Small updates!

**Pull request merged**